nullable/nullable.go

```diff
@@ -0,0 +1,137 @@
+// Package nullable provides a safe way to represent nullable primitive values in Go.
+package nullable
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/Cleverse/go-utilities/errors"
+)
+
+type primitive interface {
```

Comment:
I think this constraint should be exported.

Reply:
exported